pyproject.toml

```diff
@@ -237,6 +239,10 @@ exclude = [
     "src/otx/core/engine/utils/*",
     "src/otx/algo/classification/backbones/*",
     "for_developers/helpers.py",
+
+    # Ruff complains it but don't know how to fix since it literally showed no useful logs.
+    # https://github.com/openvinotoolkit/training_extensions/actions/runs/7176557723/job/19541622452?pr=2718#step:5:170
```

Comment:
Please let me know if someone has an idea to fix this.

Reply:
It seems that ruff already fixed errors, so you just save the modified folder and `add --> commit --> push` might resolve the error, I think.

Reply:
See my local has no problem on https://github.com/openvinotoolkit/training_extensions/pull/2718/commits/676bdbd230a5a69a3b0c82018e8c3ac9dcd37ccf
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/434ab338-3afc-4b6d-9895-1cb7e3bc31cd)

Reply:
However, CI's pre-commit raised an error but does not give any useful information: https://github.com/openvinotoolkit/training_extensions/actions/runs/7177128017/job/19543146905?pr=2718#step:5:174
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/08f3ff8f-2ada-487a-80fe-3e3448be4819)

Reply:
Hmm, it seems weird. In my experience, when ruff modified my files, he said to me like below.
![image](https://github.com/openvinotoolkit/training_extensions/assets/80735344/23227cde-1ddd-40b8-8296-5beccf34960e)
So, the only thing I need to do is add the modified files and then re-push. 

So, I'm also not sure the reason of the current issue, let's keep it as ignoring.